docs/src/components/productMaterial/MaterialStyling.tsx

```diff
@@ -311,11 +305,12 @@ export default function MaterialStyling() {
               sx={{
                 maxHeight: index === 2 ? 282 : 400,
                 overflow: 'auto',
-                position: 'relative',
               }}
             >
-              <HighlightedCode copyButtonHidden plainStyle code={code} language="jsx" />
-              <FlashCode startLine={startLine[index]} endLine={endLine[index]} sx={{ mx: 1 }} />
+              <Box sx={{ position: 'relative' }}>
```

Comment:
I'm curious why an extra DOM node here (as opposed to having `position: 'relative'` in the `Frame.Info`) is better than `sx={{ mx: 1 }}` in the `FlashCode` component?

Reply:
`FlashCode` position is set assuming the origin is the same with `HighlightedCode`. If the position relative is on `Frame.Info` the origin between the two is not the same, the padding impacts HighlightedCode but doesn't impact FlashCode.